spring-boot-admin-docs/pom.xml

```diff
@@ -98,7 +105,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-project-info-reports-plugin</artifactId>
-                <version>3.8.0</version>
+                <version>3.4.2</version>
```

Comment:
going back to an older version?

Reply:
good catch, I will check wether we need this plugin anyway.

Reply:
I will remove it entirely as we do not use mvn site goal (since ages).